tests/threshold/threshold-config-threshold-both-flow/test.yaml

```diff
@@ -0,0 +1,21 @@
+pcap: ../../http-all-headers/input.pcap
+
+requires:
+  min-version: 8
+
+args:
+- --set threshold-file=${TEST_DIR}/threshold.config
+- --simulate-ips
+
+retry: 3
```

Comment:
What is this retry ?
Flaky test ?

Reply:
The retry was first added for tests that are sensitive to timing and may need a few goes to test properly, so not really flaky, but kinda. If a test uses it, it should probably state why, or you can assume its sensitive to timing.